yarn.lock

```diff
@@ -0,0 +1,7648 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This looks like you accidentally ran `yarn install` @katelynienaber. Can delete this file?